nodedb.go

```diff
@@ -122,17 +133,32 @@ func (ndb *nodeDB) GetNode(nk []byte) (*Node, error) {
 	ndb.opts.Stat.IncCacheMissCnt()
 
 	// Doesn't exist, load.
-	buf, err := ndb.db.Get(ndb.nodeKey(nk))
```

Comment:
`nk` is the node key but without prefix?

Reply:
yes